redis/lock.py

```diff
@@ -50,6 +51,26 @@ class Lock(object):
         return 1
     """
 
+    # KEYS[1] - lock name
+    # ARGS[1] - token
+    # ARGS[2] - additional milliseconds
+    # return 1 if the locks time was extended, otherwise 0
+    LUA_EXTEND_TO_SCRIPT = """
+        local token = redis.call('get', KEYS[1])
+        if not token or token ~= ARGV[1] then
+            return 0
+        end
+        local expiration = redis.call('pttl', KEYS[1])
+        if not expiration then
+            expiration = 0
+        end
```

Comment:
I checked the document: https://redis.io/commands/pttl

I am not sure when would `pttl key` be `nil` or `false`? Is this check necessary?